### ISSUE TITLE
Can't display color in chat if it's creeper-shaped

### ISSUE THREAD

**Issue by StellarPurr** [68mutuba/FireWorks#1]
First, I want to say thank you for your awesome work. This plugin makes it possible to celebrate Chinese New Year in my server. Happy Chinese New Year!

You can see the issue in the picture below.
![Can't display color](https://user-images.githubusercontent.com/48518325/107548925-320b3480-6c0a-11eb-82f2-2c5f6e28c017.png)

Maybe here is the problem. 
https://github.com/68mutuba/FireWorks/blob/383de609ec7df6bea6cb60cda21083c9000436f4/src/io/github/mutuba/fireworks/command/FireWorks.java#L415

**Comment by 68mutuba** [comment]
Happy Chinese New Year!
First! Thank you for using Fire Works Plugin!
We hope you will continue to use it for celebrations in the future.
Second! I want to say thank you for reporting the issues.
I am very grateful to you for your report sentence, as it made it easy to find the problem.
This has been fixed in Correction of issues number 1.
Then I raised a new jar file that I just fixed.
Thank you for your continued support.
